internal/gui/gui_linux.go

```diff
@@ -0,0 +1,6 @@
+//+build linux
+
+package gui
+
+// TODO (darkrengarius): change path
+const iconPath = "/Applications/Skywire.app/icon.png"
```

Comment:
@jdknives, @i-hate-nicknames need an input on where this iconPath should be or if this should be hardcoded like this, same case in `darwin` and `windows`